6.0/gram5/admin/index.html

```diff
@@ -4,16 +4,17 @@
 <meta charset="UTF-8">
 <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="generator" content="Asciidoctor 1.5.8">
+<meta name="generator" content="Asciidoctor 1.5.6.1">
```

Comment:
Just noticed that we are using different AsciiDoctor versions. That implies additional changes to the HTML files not related to the actual (content) changes to the ADOC files, i.e. styling, comments, etc.. Even when #4 is fixed, this could still happen when AsciiDoctor on the CI instances gets updated.